src/fhirdefs/load.ts

```diff
@@ -75,33 +74,33 @@ export async function loadDependency(
       }
       // if the date on the package.manifest.json does not match the date on the cached package
       // set the packageUrl to trigger a re-download of the package
+      const dateRegex = /(\d{4})(\d{2})(\d{2})(\d{2})(\d{2})(\d{2})/;
+      const dateFormatString = '$1-$2-$3T$4:$5:$6';
       if (manifest?.data?.date !== cachedPackageJSON?.date) {
         packageUrl = `${igUrl}/package.tgz`;
         if (cachedPackageJSON) {
           logger.debug(
-            `Cached package date for ${fullPackageName} (${moment(
-              cachedPackageJSON.date,
-              'YYYYMMDDHHmmss'
-            ).format(
-              'YYYY-MM-DDTHH:mm:ss'
-            )}) does not match last build date on build.fhir.org (${moment(
-              manifest?.data?.date,
-              'YYYYMMDDHHmmss'
-            ).format('YYYY-MM-DDTHH:mm:ss')})`
+            `Cached package date for ${fullPackageName} (${cachedPackageJSON.date?.replace(
+              dateRegex,
+              dateFormatString
```

Comment:
You're probably ready for this PR to just be done, but I feel like it would be cleaner to put the regex and replacement logic in a `formatDate` function so that in the four places you use it, you can just do it like `${formatDate(cachedPackageJSON.date)}`.  Nit-picky, maybe -- but it would make these long debug statements easier to read.